tasks/request_files/schemas/input.json

```diff
@@ -0,0 +1,54 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://github.com/nasa/cumulus-orca/blob/master/tasks/request_files/schemas/input.json",
+  "title": "request_files Lambda Input",
+  "description": "The input for the request_files Lambda.",
+  "type": "object",
+  "properties": {
+    "job_id": {
+      "description": "The unique identifier used for tracking requests. If not present, will be generated.",
+      "type": "string"
+    },
+    "granules": {
+      "description": "A list of objects representing individual files to restore.",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "granuleId": {
+            "description": "The id of the granule being restored.",
+            "type": "string"
+          },
+          "keys": {
+            "description": "Keys related to where the files will be located.",
+            "type": "array",
+            "items": {
+              "type": "object",
+              "properties": {
+                "key": {
+                  "description": "Name of the file within the granule.  # TODO: This or example lies.",
```

Comment:
I think the TODO is in the wrong spot. I feel it should be at the array of granules level. I believe only 1 granule is ever passed but a granule can have multiple files (keys). I could be wrong though. Maybe not possible with this first go around but we should maybe look at normalizing some of the key names in future work like keys -> files for example.

Reply:
Check out my todos in inner_task. Somebody wrote the code to loop over an array of granules, but then only ACTUALLY processes the last granule in the list. This particular todo is in regards to the example that uses a file path, despite this 'key' stating that it is a file name.